### PR TITLE
chore: shuffle e2e tests and add first conent-type

### PR DESCRIPTION
**Pull request by joshuaellis**
### What does it do?

* Moves tests into domain spaces
* Adds a dummy content type
* Adds a dummy test for the Content-Manager

### Why is it needed?

* So we can look at running multiple test apps in parallel based on the domain

### Related issue(s)/PR(s)

* resolves CONTENT-1554

### CODE REVIEW

File: playwright.config.js [hunk 1]
```diff
@@ -95,7 +95,7 @@ const config = {
 
   /* Run your local dev server before starting the tests */
   webServer: {
-    command: 'cd test-apps/playwright && yarn start',
+    command: 'cd test-apps/playwright && yarn develop',
```
Comment (Marc-Roig) [inline playwright.config.js:98]: should we test with yarn develop?

Comment (joshuaellis) [reply playwright.config.js:98]: So i thought no to begin with, but then realised we can't test the CTB without it 🙃

Comment (Marc-Roig) [reply playwright.config.js:98]: true 😓

**Review by Marc-Roig — APPROVED** [review]
e2e tests are coming 💃

**Pull request merged**